opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -362,10 +360,12 @@ impl Debug for UserEventsExporter {
 
 impl opentelemetry_sdk::logs::LogExporter for UserEventsExporter {
     async fn export(&self, batch: opentelemetry_sdk::logs::LogBatch<'_>) -> OTelSdkResult {
+        let mut res = Ok(());
         for (record, instrumentation) in batch.iter() {
-            let _ = self.export_log_data(record, instrumentation);
+            // no, res is not overwritten as the batch will only have one item
```

Comment:
I think we can remove the no here 😁

```suggestion
            // res is not overwritten as the batch will only have one item
```

Reply:
I liked the no actually  jk :)